src/main/java/seedu/address/model/tier/Tier.java

```diff
@@ -8,8 +8,7 @@
  * Guarantees: immutable; name is valid as declared in {@link #isValidTierName(String)}
  */
 public class Tier {
-
-    public static final String MESSAGE_CONSTRAINTS = "Tier can be set to 'gold', 'silver', 'bronze', 'reject' or 'na'. "
+    public static final String MESSAGE_CONSTRAINTS = "Tier must be set to Gold, Silver, Bronze, Reject, or NA. "
```

Comment:
I think we should standardize the quotes for Tier and Status.

Reply:
Nice catch!